include/onnxruntime/core/session/onnxruntime_session_options_config_keys.h

```diff
@@ -61,6 +61,21 @@ static const char* const kOrtSessionOptionsEnableQuantQDQCleanup = "session.enab
 // GeluApproximation has side effects which may change the inference results. It is disabled by default due to this.
 static const char* const kOrtSessionOptionsEnableGeluApproximation = "optimization.enable_gelu_approximation";
 
+#ifdef ENABLE_TRAINING
+// Specifies a list of op types for memory footprint reduction.
+// The value should be a ","-delimited list of pair of
+// <subgraph string : alleviation strategy : number of subgraph to apply>.
+// For example, "Gelu+Cast+:1:0,Dropout:1:1".
+//   "alleviation strategy" currently has valid values: 0 - disabled, 1 - recompute.
+//   "number of subgraph to apply" is used to control how many subgraphs to apply alleviation, to avoid "oversaving"
```

Comment:
I don't understand "number of subgraph to apply" : Can you explain more.

Reply:
yeah currently it's top 3, I had the comments in the transformer code.

Reply:
while I had an idea to enable them with stride, saying there are 6, stride is 1, then we enable layer 0, 2, 4 which maybe do some good on memory fragementation. I did not add it now, but will do more experiments to see whether it helps or not.